docker/run

```diff
@@ -100,6 +100,10 @@ compose_recreate() {
     echo "\nRecreating $c container ..."
     docker_compose stop $c
     docker_compose rm -f $c
+    # check conainer have named volume
+    if docker volume list | grep -E "$c-$ENV-*" 2>&1 >/dev/null ; then
+      docker volume rm $(docker volume list | grep -E "$c-$ENV-*" | awk '{print $2}') 2>&1 >/dev/null
```

Comment:
hmm, doesn't do `docker-compose down -v` do the trick for you?

Reply:
```
docker-compose rm --help
....
-v            Remove any anonymous volumes attached to containers
...
```

will not work with named volumes.

Reply:
I'm talking about `docker-compose down -v`:

```
...
-v, --volumes           Remove named volumes declared in the `volumes`
                            section of the Compose file and anonymous volumes
                            attached to containers.
...
```

Reply:
I mean `docker-compose dow` bring all service down. You can't select a single service. 
This function will `stop` a singel service.

Reply:
good point 🙂 👍